features/networking/sctp.feature

```diff
@@ -18,6 +18,13 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[0].name %> |
     Then the step should succeed
     And the pod named "sctpserver" becomes ready
+    And I wait up to 20 seconds for the steps to pass:
```

Comment:
Thanks @weliang1 for investigating this. However I suggest to avoid landing in similar situation like #1643 in future , we can use just use ```nc``` instead of absolute path. So Instead of ```Then the output should contain "/usr/bin/nc"``` we can opt for ```Then the step should succeed``` I guess the later will return exit code 0 based on successful execution. WDYT and if you opt for that you might wanna change ```/usr/bin/nc``` references to just ```nc``` in later steps. I suggest you try this with just one testcase and let us know if that seems okay.

Reply:
@anuragthehatter Thanks for good suggestions, update both places and testing got passed.

And I wait up to 20 seconds for the steps to pass:                    # features/step_definitions/meta_steps.rb:33
      [21:37:19] INFO> Shell Commands: oc exec sctpclient  --kubeconfig=/home/weliang/workdir/weliang-weliang/ocp4_testuser-0.kubeconfig -n 34mz7  -i -- bash -c which\ nc
      
      STDERR:
      which: no nc in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
      E1116 16:37:19.984544   23320 v2.go:105] write tcp 10.10.116.251:34982->10.0.102.163:6443: use of closed network connection
      command terminated with exit code 1
      
      [21:37:19] INFO> Exit Status: 1
      [21:37:21] INFO> Shell Commands: oc exec sctpclient  --kubeconfig=/home/weliang/workdir/weliang-weliang/ocp4_testuser-0.kubeconfig -n 34mz7  -i -- bash -c which\ nc
      
      STDERR:
      which: no nc in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
      E1116 16:37:21.686011   23335 v2.go:105] write tcp 10.10.116.251:34986->10.0.102.163:6443: use of closed network connection
      command terminated with exit code 1
      
      [21:37:21] INFO> Exit Status: 1
      [21:37:22] INFO> Shell Commands: oc exec sctpclient  --kubeconfig=/home/weliang/workdir/weliang-weliang/ocp4_testuser-0.kubeconfig -n 34mz7  -i -- bash -c which\ nc
      /usr/bin/nc
      
      STDERR:
      E1116 16:37:23.419988   23357 v2.go:105] write tcp 10.10.116.251:34990->10.0.102.163:6443: use of closed network connection
      
      [21:37:23] INFO> Exit Status: 0
      """
      When I execute on the "sctpclient" pod:
        | bash | -c | which nc    |
      Then the step should succeed
      """
      # sctpserver pod start to wait for sctp traffic
    When I run the :exec background client command with:                  # features/step_definitions/cli.rb:13
      [21:37:23] INFO> Shell Commands: oc exec sctpserver  --kubeconfig=/home/weliang/workdir/weliang-weliang/ocp4_testuser-0.kubeconfig --namespace=34mz7  -- sh -c nc\ -l\ 30102\ --sctp\ -k\ -m\ 5\ -w\ 90
      | pod              | sctpserver                       |
      | namespace        | <%= project.name %>              |
      | oc_opts_end      |                                  |
      | exec_command     | sh                               |
      | exec_command_arg | -c                               |
      | exec_command_arg | nc -l 30102 --sctp -k -m 5 -w 90 |
    Then the step should succeed                                          # features/step_definitions/common.rb:4
waiting for operation up to 3600 seconds..
      # sctpclient pod start to send sctp traffic
    And I wait up to 60 seconds for the steps to pass:                    # features/step_definitions/meta_steps.rb:33
      [21:37:24] INFO> Shell Commands: oc exec sctpclient  --kubeconfig=/home/weliang/workdir/weliang-weliang/ocp4_testuser-0.kubeconfig --namespace=34mz7  -- sh -c echo\ test-openshift\ \|\ nc\ -v\ 10.128.2.175\ 30102\ --sctp
      
      STDERR:
      Ncat: Version 7.70 ( https://nmap.org/ncat )
      Ncat: Connected to 10.128.2.175:30102.
      Ncat: 15 bytes sent, 0 bytes received in 0.01 seconds.